extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -430,14 +431,14 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		// the top we'll expand nodes until we find the parent of the Notebook we're looking for
 		// get the children of root node and expand the nodes to the notebook level.
 		await this.getChildren(parentBook.rootNode);
-		// The path to the parent of the Notebook we're looking for (this is the node we're looking to expand)
-		const parentPath = notebookPath.substring(0, notebookPath.lastIndexOf(path.posix.sep));
+		// The path to the Notebook we're looking for (these are the nodes we're looking to expand)
```

Comment:
This isn't the path to the Notebook we're looking for so I think the comment could be updated better. And given that you're changing how the logic works I would say you should update all these comments to be accurate (since now we're actually walking up the path from the containing folder and checking each folder one by one)